src/site/content/en/blog/codelab-building-a-sidenav-component/index.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: codelab
+title: "Codelab: Building a Sidenav component"
+authors:
+  - adamargyle
+description: |
+  Learn how to build a responsive slide out side navigation layout component.
+date: 2020-11-25
+hero: hero.jpg
+thumbnail: thumb.jpg
```

Comment:
```suggestion
```
Hero and thumbnail pages have no effect when we use codelab layout, right? Please remove the hero image from this directory.

Reply:
here's why i add them, which reminds me, the centering post needs the fix too.

![image](https://user-images.githubusercontent.com/1134620/104659573-b608e580-5679-11eb-880e-2140d460aa77.png)